adam-core/src/test/scala/org/bdgenomics/adam/rdd/features/FeatureRDDFunctionsSuite.scala

```diff
@@ -0,0 +1,256 @@
+    val features = sc.loadBed(inputPath)
+    val outputPath = tempLocation(".gtf")
+    features.saveAsGtf(outputPath)
+  }
+
+  sparkTest("save BED as BED format") {
+    //val inputPath = resourcePath("features/gencode.v7.annotation.trunc10.bed")
+    val inputPath = resourcePath("features/ItemRGBDemo.bed")
+    val features = sc.loadBed(inputPath)
+    val outputPath = tempLocation(".bed")
+    features.saveAsBed(outputPath)
+  }
+
+  sparkTest("save BED as IntervalList format") {
+    //val inputPath = resourcePath("features/gencode.v7.annotation.trunc10.bed")
+    val inputPath = resourcePath("features/ItemRGBDemo.bed")
+    val features = sc.loadBed(inputPath)
+    val outputPath = tempLocation(".interval_list")
+    features.saveAsIntervalList(outputPath)
+  }
+
+  sparkTest("save BED as NarrowPeak format") {
+    //val inputPath = resourcePath("features/gencode.v7.annotation.trunc10.bed")
+    val inputPath = resourcePath("features/ItemRGBDemo.bed")
+    val features = sc.loadBed(inputPath)
+    val outputPath = tempLocation(".narrowPeak")
+    features.saveAsNarrowPeak(outputPath)
+  }
+
+  sparkTest("round trip BED format") {
+    //val inputPath = resourcePath("features/gencode.v7.annotation.trunc10.bed")
+    val inputPath = resourcePath("features/ItemRGBDemo.bed")
+    val expected = sc.loadBed(inputPath)
+    val outputPath = tempLocation(".bed")
+    expected.saveAsBed(outputPath)
+
+    // grab all partitions, may not necessarily be in order; sort by reference
+    val actual = sc.loadBed(outputPath + "/part-*")
+    val pairs = sortByReference(expected).zip(sortByReference(actual)).collect
+
+    // separate since assert is not serializable
+    pairs.foreach({ pair: (Feature, Feature) => assert(pair._1 === pair._2) })
+  }
+
+  sparkTest("save IntervalList as GTF format") {
+    val inputPath = resourcePath("features/SeqCap_EZ_Exome_v3.hg19.interval_list")
+    val features = sc.loadIntervalList(inputPath)
+    val outputPath = tempLocation(".gtf")
+    features.saveAsGtf(outputPath)
+  }
+
+  sparkTest("save IntervalList as BED format") {
+    val inputPath = resourcePath("features/SeqCap_EZ_Exome_v3.hg19.interval_list")
+    val features = sc.loadIntervalList(inputPath)
+    val outputPath = tempLocation(".bed")
+    features.saveAsBed(outputPath)
+  }
+
+  sparkTest("save IntervalList as IntervalList format") {
+    val inputPath = resourcePath("features/SeqCap_EZ_Exome_v3.hg19.interval_list")
+    val features = sc.loadIntervalList(inputPath)
+    val outputPath = tempLocation(".interval_list")
+    features.saveAsIntervalList(outputPath)
+  }
+
+  sparkTest("save IntervalList as NarrowPeak format") {
+    val inputPath = resourcePath("features/SeqCap_EZ_Exome_v3.hg19.interval_list")
+    val features = sc.loadIntervalList(inputPath)
+    val outputPath = tempLocation(".narrowPeak")
+    features.saveAsNarrowPeak(outputPath)
+  }
+
+  /*
+  sparkTest("round trip IntervalList format") {
```

Comment:
Eliminate dead test code, or change to an ignored test --> `ignore("round trip..."){`